src/lib/y2partitioner/actions/controllers/md.rb

```diff
@@ -173,18 +187,26 @@ def available?(partition)
           mount.nil? || mount.empty?
         end
 
+        def min_chunk_size
+          [default_chunk_size, Y2Storage::DiskSize.KiB(64)].min
+        end
+
+        def max_chunk_size
+          Y2Storage::DiskSize.MiB(64)
+        end
+
         def default_chunk_size
           case md.md_level.to_sym
           when :raid0
-            Y2Storage::DiskSize.KiB(32)
+            Y2Storage::DiskSize.KiB(64)
           when :raid1
             Y2Storage::DiskSize.KiB(4)
           when :raid5, :raid6
             Y2Storage::DiskSize.KiB(128)
           when :raid10
-            Y2Storage::DiskSize.KiB(32)
+            Y2Storage::DiskSize.KiB(64)
           else
-            Y2Storage::DiskSize.KiB(4)
+            Y2Storage::DiskSize.KiB(64)
```

Comment:
I see four times in code 64. What about using constant with reference to this bug explaining why now(!) 64kb is reasonable minimum, so we know it for future?

Reply:
Those values seem to fall from the sky. There is not exactly any reference we could point to. Just see the PBI.